src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import propTypes from "prop-types";
+import { useDispatch } from "react-redux";
+
+import styles from "components/task/create-new/index.module.scss";
+import Button from "components/base/button";
+import TextArea from "components/base/text-area";
+import Image from "components/base/image";
+import { addNewTaskSchema } from "utils/schema";
+import { addTaskToTodo } from "store/actions/todo";
+import { TASK_TEXTAREA_NUM_OF_ROW } from "utils/const";
+import { generateTaskObject } from "utils/helper";
+import { DELETE_ICON_ALT_TAG } from "utils/const/images";
```

Comment:
Lets not scatter consts like this. we can import from one generic file for consts. In that file we can import the consts from other files and export them.
```suggestion
import { TASK_TEXTAREA_NUM_OF_ROW, DELETE_ICON_ALT_TAG } from "utils/const";
```